QUARKUS-343.md

```diff
@@ -0,0 +1,48 @@
+# QUARKUS-343 - Quarkus and non JSON payload in REST endpoints
+
+## Scope of the testing
+The test scope includes validating Quarkus's capability to handle non-JSON payloads (XML, YAML, and images) efficiently in REST endpoints.
+Tests will cover both JVM and Native mode executions.
+#### Test scenarios
+
+        1. XML Payload Scenario:
+            * Verify the endpoint's ability to consume and produce XML payloads.
+            * Send a request with an XML payload to the endpoint.
+            * Validate the response, ensuring it matches the expected XML structure.
+    
+        2. YAML Payload Scenario:
+            * Check the endpoint's ability to return YAML from a YAML file.
+            * Send a request with a YAML payload from a file to an endpoint.
+            * Validate the response, ensuring it matches the expected YAML content.
+    
+        3. Image Payload Scenario:
+            * Verify the endpoint's ability to return an image by making a GET request.
```

Comment:
I don't think any Quarkus endpoint has ability to return image. Can you be more specific? I think you should safe in which form you return it, like byte etc. In which case there is probably no difference to that data types for other non-images. Anyway, I 'd appreciate you to be specific because I presume there is more than one way to do that and I want to know which way are you going to test and why.